docs/api-reference/json.rst

```diff
@@ -167,7 +167,9 @@ Project
                     "yanked_reason": null
                 }
             ],
-            "vulnerabilities": []
+            "vulnerabilities": [],
+            "critical_project": False,
+            "requires_2fa": False
```

Comment:
Most services don't publish if an individual user has 2FA. While this doesn't _directly_ tell you that, it does sort of strongly hint at it. Does it make sense to hold this back and only expose if a project is critical?

Reply:
As a consumer of this data/security team person I agree its valuable, I'm just sort of predicting that this might upset people.

Reply:
I can drop it for now, we can always add it back in later.